google/cloud/forseti/notifier/notifiers/findings.py

```diff
@@ -52,7 +52,7 @@ def _transform_to_findings(violations):
                     'resource_type': violation.get('resource_type'),
                     'resource_id': violation.get('resource_id'),
                     'rule_index': violation.get('rule_index'),
-                    'inventory_index_id': violation.get('inventory_index_id'),
+                    'scanner_index_id': violation.get('scanner_index_id'),
```

Comment:
It's okay to add the `scanner_index_id` here.  But we still should keep the `inventory_index_id` reference because it will help the user to know right away, which inventory the violation is coming from, without having to do another lookup.

Reply:
Done.